index.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport"
+      content="width=device-width, initial-scale=1">
+    <title>Works</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+   
+    <!-- container-s -->
+    <div class="container sp">
+
+      <!--  wrapper-s -->
+      <div class="wrapper">
+
+        <div class="grid column row">
+          <div class="one"> 
+          </div>
+          <div>Tonic</div>
+          <div>
+            <div>CANOPY</div>
+            <ul class="flex">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div>
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </div>
+          <div>
+            <ul class="flex">
+              <li>
+                html
+              </li>
+              <li>
+                css
+              </li>
+              <li>
+                javascript
+              </li>
+            </ul>
+          </div>
+          <div>
+            <button> See Project</button>
```

Comment:
you border color is not the same as the one in the Figma design, I suggest implementing that 

- [ ] Your button 
![image](https://user-images.githubusercontent.com/61790587/164916952-36b49ddf-8899-4306-8179-ed429766028d.png)

- [ ] button in Figma design 
![image](https://user-images.githubusercontent.com/61790587/164916975-52944262-0c25-46c6-bd77-7a7750858648.png)